lib/clientOptions.ts

```diff
@@ -0,0 +1,9 @@
+export interface ClientOptions {
+  apiHostname?: string;
+  defaultVersion?: string;
+}
+
+export const ClientOptionsDefaults: ClientOptions = {
+  apiHostname: "https://app.procore.com",
```

Comment:
It may produce the `CORS policy` error:

`Access to fetch at 'https://app.procore.com/rest/v1.0/me?' from origin 'http://localhost:4050' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: The value of the 'Access-Control-Allow-Credentials' header in the response is '' which must be 'true' when the request's credentials mode is 'include'.`